### PR TITLE
Upgrade the addon with the latest ucc

### PR DESCRIPTION
**Pull request by hardikhdholariya**
- Updated the js and python libs as per the latest UCC version 5.28.1
- Updated the manifest.json with the latest schema v2.0.0 and added the required info for the deployment and target workloads
- Added sc_admin role to the default.meta file.

### CODE REVIEW

**Review by VatsalJagani — APPROVED** [review]
Everything looks okay as far as we are able to configure the Add-on Input from UI and able to collect the data.

App version needs to be changed.

README file needs to be updated.

File: TA-microsoft-graph-security-score/default/addon_builder.conf
Comment (mahirchavda) [inline TA-microsoft-graph-security-score/default/addon_builder.conf:1]: Lets remove this file as we are using UCC now.

File: TA-microsoft-graph-security-score/appserver/static/js/build/globalConfig.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
         "version": "1.0.1",
         "apiVersion": "3.0.0",
```
Comment (mahirchavda) [inline TA-microsoft-graph-security-score/appserver/static/js/build/globalConfig.json]: Update apiVersion to the UCC version that we used

Comment (hardikhdholariya) [reply TA-microsoft-graph-security-score/appserver/static/js/build/globalConfig.json]: Updated with version 5.28.1

**Review by mahirchavda — APPROVED** [review]

**Pull request merged**